src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/FormPass.php

```diff
@@ -53,11 +54,15 @@ public function process(ContainerBuilder $container)
         $typeExtensions = array();
 
         foreach ($container->findTaggedServiceIds('form.type_extension') as $serviceId => $tag) {
-            $alias = isset($tag[0]['alias'])
-                ? $tag[0]['alias']
-                : $serviceId;
+            $extendedType = $serviceId;
+            if (isset($tag[0]['extended_type'])) {
```

Comment:
Option names are normalized: https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php#L240

I really don't like the `alias` name:

 * It's not clear that this is the alias of the form it extends
 * If that's clear, it makes no sense anymore as there is no alias anymore (it's a FQCN from now on)